impy/models/__init__.py

```diff
@@ -0,0 +1,10 @@
+from impy.models.sophia import Sophia20
+from impy.models.sibyll import Sibyll21, Sibyll23, Sibyll23c, Sibyll23c00, Sibyll23c01,\
+    Sibyll23c02, Sibyll23c03, Sibyll23c04, Sibyll23d
+from impy.models.dpmjetIII import DpmjetIII191, DpmjetIII192, DpmjetIII306
+from impy.models.epos import EposLHC
+from impy.models.phojet import Phojet112, Phojet191
+from impy.models.urqmd import UrQMD34
+from impy.models.qgsjet import QGSJet01c, QGSJetII03, QGSJetII04
+from impy.models.pythia6 import Pyphia6
+from impy.models.pythia8 import Pyphia8
```

Comment:
Is there some notable initialization cost when these models are imported? If yes, then `__init__.py` should not all import them. For any given run, I will mostly only need one of these models, but the way it is written here, I always have to pay for the initiation cost of all of them.

Reply:
All that imports are needed only because it allows to have autocompletion like:

```python
import impy
impy.models. # <- here the autocompletion gives the list of available models
```

The initialisation time is around 1 - 1.5 sec. Interestingly, itt doesn't depend on number of imports, only on the fact that there is an import in __init__.py file. The cost of initialisation like:
```python
from impy.models.sophia import Sophia20
```
is around 0.01 sec. 

I don't know whether 1 sec worth it to sacrifice the convenience of the autocompletion.

Reply:
Ok, fine!